files/spec/cookbook_spec.rb

```diff
@@ -60,6 +60,10 @@ def run_chef(args)
 #     -   set x to "16" \(default value\)
 #     -   set y to 4 \(default value\)
 # /)
+
+    if Chef::VERSION.to_f >= 12.1
+      # expect(result.stdout).to match /ResourceBuilder superclass: Chef::ResourceBuilder/
```

Comment:
See lines 41-61. It seems to be pretty common in this cookbook to comment out tests that don't run correctly.